### PR TITLE
Billbl calling meeting apis

### PR DESCRIPTION
**Pull request by billbliss**
Inititial checkin of calling and meeting APIs

### CODE REVIEW

File: api-reference/beta/api/application_post_calls.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
+# Create call
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+Use this API to create a new call.
```
Comment (Lauragra) [inline api-reference/beta/api/application_post_calls.md]: You're missing the return after the beta note; please add.

File: api-reference/beta/api/application_post_calls.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+| Permission type                        | Permissions (from least to most privileged) |
+|:---------------------------------------|:--------------------------------------------|
+| Delegated (work or school account)     |                                             |
+| Delegated (personal Microsoft account) |                                             |
+| Application                            |                                             |
+
```
Comment (Lauragra) [inline api-reference/beta/api/application_post_calls.md]: Need to complete the permissions info.

File: api-reference/beta/api/application_post_calls.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+  "tenantId": "tenantId-value",
+  "terminationReason": "terminationReason-value",
+  "toneInfo": {
+    "sequenceId": 99,
+    "tone": "tone0"
+  }
+}
+```
+
+In the request body, supply a JSON representation of [call](../resources/call.md) object.
```
Comment (Lauragra) [inline api-reference/beta/api/application_post_calls.md]: We've already said this in the Request section so no need to repeat in the example.

File: api-reference/beta/api/application_post_calls.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+
+##### Response
+Here is an example of the response. Note: The response object shown here may be truncated for brevity. All of the properties will be returned from an actual call.
```
Comment (Lauragra) [inline api-reference/beta/api/application_post_calls.md]: Ideally, we revise the note text as follows:

>**Note**: The response object shown here might be shortened for readability. All the properties will be returned from an actual call.

File: api-reference/beta/api/audioRoutingGroup_delete.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+﻿# Delete audioRoutingGroup
+
```
Comment (Lauragra) [inline api-reference/beta/api/audioRoutingGroup_delete.md]: Generally in titles, we use the friendly name of the resource: Delete audio routing group.

File: api-reference/beta/api/audioroutinggroup_get.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
+# Get audioRoutingGroup
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+Retrieve the properties and relationships of audioRoutingGroup object.
+
```
Comment (Lauragra) [inline api-reference/beta/api/audioroutinggroup_get.md]: Fix the return after the note, and update the title to use friendly name.

File: api-reference/beta/api/audioroutinggroup_get.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+| Permission type                        | Permissions (from least to most privileged) |
+|:---------------------------------------|:--------------------------------------------|
+| Delegated (work or school account)     |                                             |
+| Delegated (personal Microsoft account) |                                             |
+| Application                            |                                             |
+
```
Comment (Lauragra) [inline api-reference/beta/api/audioroutinggroup_get.md]: Missing the permissions.

File: api-reference/beta/api/audioroutinggroup_get.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+GET /app/calls/{id}/audioRoutingGroups/{id}
+GET /applications/{id}/calls/{id}/audioRoutingGroups/{id}
+```
+
+## Optional query parameters
+This method supports the [OData Query Parameters](http://graph.microsoft.io/docs/overview/query_parameters) to help customize the response.
```
Comment (Lauragra) [inline api-reference/beta/api/audioroutinggroup_get.md]: We need to use the new URL for the query parameters topic: https://developer.microsoft.com/en-us/graph/docs/concepts/query_parameters

File: api-reference/beta/api/application_post_calls.md [hunk 9]
```diff
@@ -0,0 +1,251 @@
...
+| Delegated (personal Microsoft account) |                                             |
+| Application                            |                                             |
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+POST /app/calls
+POST /applications/{id}/calls
+```
+
```
Comment (Lauragra) [inline api-reference/beta/api/application_post_calls.md]: Be sure to include the Optional query parameters H2 consistently in all API topics.

File: api-reference/beta/api/application_post_calls.md (hunk 2 above)
Comment (kanchm) [reply api-reference/beta/api/application_post_calls.md]: We will update the permissions across the board for our APIs.

File: api-reference/beta/api/application_post_calls.md (hunk 9 above)
Comment (kanchm) [reply api-reference/beta/api/application_post_calls.md]: We have it on 'get' operations.  Do you need them on all?

**Comment by billbliss** [comment]
Closed in favor of PR #3095

**Pull request closed without merging**